Project - Linghan Zheng

Leeann954 opened the pull request:
It's my first draft on Github.

Enchufa2 reviewed (changes requested):
Thanks, great work. But the idea is that the file called `_projects/2024/100540803/100540803.Rmd` should be your post, so that it's integrated in the webpage's projects section. So could you please move the contents of `first version.qmd` to the file above and remove this qmd as well as `first-version_files` and `first version_files`? If you need the `images` directory, that should be moved under `_projects/2024/100540803` too.

Leeann954 commented:
Hi Inaki,

I have already moved the file to _projects/2024/100540803/100540803.Rmd and deleted the original first version. However, I'm now encountering this conflict message regarding projects.html, projects/projects.json, and sitemap.xml. Could you help me resolve this issue or guide me on the next steps to take?

Thanks!
![image](https://github.com/user-attachments/assets/63aafeec-10b5-48e3-8de3-fb7a009bb50f)

Leeann954 commented:
Hi Iñaki,

I have already moved the file to _projects/2024/100540803/100540803.Rmd and deleted the original first version. However, I'm now encountering this conflict message regarding projects.html, projects/projects.json, and sitemap.xml. Could you help me resolve this issue or guide me on the next steps to take?

Thanks!
![image](https://github.com/user-attachments/assets/f2d85409-a7c9-4880-8bf6-f754f313cf2b)

Enchufa2 commented on _projects/2024/100540803/100540803.Rmd:
```diff
@@ -0,0 +1,601 @@
+---
+title: "Global Fish Stock Overfishing Status"
+description: "Replication of a chart illustrating fish capture and abundance relative to sustainable levels."
+author: 
+  name: "Linghan Zheng"
+  date: "Dec.24,2024"
+format: html
+editor: visual
```
Please, set the header as indicated in the [tutorial](https://csslab.uc3m.es/dataviz/tutorials/project/#adding-your-own-article), where it says "the header should look like this"...

Enchufa2 commented on _projects/2024/100540803/100540803.Rmd:
```diff
@@ -0,0 +1,601 @@
+
+## Original graph
+
+![Kobe Plot 1](images/picture1.png){width="100%"} ![Kobe Plot 2](images/picture2.png){width="100%"} ![Kobe Plot3](images/picture3.png){width="100%"} *Note: The original graph can be found on the [World Bank SDG Atlas - Goal 14: Life Below Water](https://datatopics.worldbank.org/sdgatlas/goal-14-life-below-water/?lang=en#c14s1).*
```
Please set the images as "external" as e.g. David does [here](https://github.com/IBiDat/dataviz/blob/81084e40371bd8161998806e9c7471e04ad47cfe/_projects/2024/100535712/100535712.Rmd#L26).

Enchufa2 commented on _projects/2024/100540803/100540803.Rmd:
````diff
@@ -0,0 +1,601 @@
+The project begins with importing the necessary datasets. The read.csv function from the base R package is used for loading CSV files.
+
+```{r}
+data <- read.csv("C://Users/zlh/Downloads/goal14.kobe.csv")
````
Please add the data here too, and reference it directly instead of using the path in your computer.

Enchufa2 commented on _projects/2024/100540803/100540803.Rmd:
```diff
@@ -0,0 +1,601 @@
+
+## Final graph
+
+![Final graph](images/picture4.png){width="100%"}
```
Instead of adding the static file, just print `p5` above, as you with the previous ones.

Enchufa2 commented on _projects/2024/100540803/100540803.Rmd:
````diff
@@ -0,0 +1,601 @@
+
+The fourth plot, created using `ggplot2`, is a static Kobe Plot that includes the following features: orange circular points representing valid data, light-colored quadrant backgrounds sampled from the original plot, two black dividing lines (x=1 and y=1), quadrant labels with light gray text (e.g., "Overfished and currently overfishing"), highlighted special points with orange fill, white outer  ...
+
+```{r,message=FALSE, warning=FALSE}
````
Please add `preview=TRUE`, so that this one is highlighted in the gallery.

Enchufa2 reviewed (changes requested):
Nice, thanks for the work with ggplot2! Some minor comments below. And don't worry about the conflicts, I'll deal with that.

Leeann954 commented:
Hi Iñaki,
I’ve implemented all the requested changes and addressed the feedback. Please review the updates and let me know if there’s anything else that needs further adjustment. Thank you!

Enchufa2 commented:
Congrats, here you are: https://csslab.uc3m.es/dataviz/projects/2024/100540803/
